docs/source/format/CanonicalExtensions.rst

```diff
@@ -283,6 +283,28 @@ UUID
    A specific UUID version is not required or guaranteed. This extension represents
    UUIDs as FixedSizeBinary(16) with big-endian notation and does not interpret the bytes in any way.
 
+8-bit Boolean
```

Comment:
nit: maybe a separate PR to just add the specification?

Reply:
This is a good idea. @pitrou suggested the same in the mailing list. I opened #43323 with only the Go commits and just rebased this PR to only include the updates to the specification.